com.unity.shadergraph/Editor/GraphUI/GraphElements/ModelUI/Blackboard/ShaderGraphSectionModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEditor.GraphToolsFoundation.Overdrive.BasicModel;
+
+namespace UnityEditor.ShaderGraph.GraphUI
+{
+    public class ShaderGraphSectionModel : SectionModel
```

Comment:
In theory we could remove this rn cause it doesnt do anything but we might need it later, so leaving it here for now